package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "java-cookie",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "bower": {
+      "version": "1.8.8",
+      "resolved": "https://registry.npmjs.org/bower/-/bower-1.8.8.tgz",
+      "integrity": "sha512-1SrJnXnkP9soITHptSO+ahx3QKp3cVzn8poI6ujqc5SeOkg5iqM1pK9H+DSc2OQ8SnO0jC/NG4Ur/UIwy7574A==",
+      "dev": true
+    }
+  }
+}
```

Comment:
Some node versions don't require package-lock so it may feel useless to some dev environments. Can you add an "engines" property in package.json to document the node version you're using to run NPM? This way myself and other maintainers know which version we should use for this project :)

Reply:
The node and npm version are actually specified in the pom.xml. Currently they are not used I think, but the globally install node/npm. We should change that, so that the globally installed version does not matter.

Reply:
Probably, if we can make the tests to always use a specific version that's better. If not, we can document the NodeJS version that the system supports for dev environment. This way we're all using the same version or `>= the same version`

Reply:
I checked again - the version specified in the `pom.xml` for the `frontend-maven-plugin` is explicitly used for everything, so it should be independent from the dev environment.